src/api/data-structure-creator.js

```diff
@@ -0,0 +1,100 @@
+export class DataStructureCreate {
+    constructor(name) {
+        if (!name) {
+            throw new Error('name is required.');
+        } else if (_isNotString(name)) {
+            throw new Error('name should be string');
+        } else {
+            this.name = name;
+        }
+        this.items = JSON.parse(localStorage.getItem(`${name}`) || "[]");
+    }
+    
+    createUser(data) {
+        let items = this.items;
+        let name = this.name;
+        if (!_isObject(data)) {
+            throw new Error('data should be an object.');
+        }
+        Object.defineProperty(data, '_id', {
+            enumerable: true,
+            configurable: false,
+            writable: false,
+            value: _generateId()
+        });
+        _validateUser(data);
+        items.push(data);
+        localStorage.setItem(name, JSON.stringify(items));
+        return data;
+    }
+
+    getUser(id) {
+        let items = this.items;
+        let user = {};
+        items.forEach((item) => {
```

Comment:
```suggestion
        items.forEach(item => {
```
single param arrow functions don't need to include the wrapping parens